src/main/webapp/app/exercises/shared/result/result-detail.component.html

```diff
@@ -17,9 +17,16 @@ <h4 class="modal-title">Feedback</h4>
             <div *ngIf="showScoreChart && result.participation?.exercise" class="result-score-chart">
                 <span class="result-score-chart-title"
                     ><span jhiTranslate="artemisApp.result.score">Score</span>: {{ result.score }}%
-                    <span *ngIf="resultIsPreliminary()" class="badge badge-warning" [ngbTooltip]="'artemisApp.result.preliminaryTooltip' | translate">{{
-                        'artemisApp.result.preliminary' | translate | uppercase
-                    }}</span>
+                    <span
+                        *ngIf="resultIsPreliminary()"
+                        class="badge badge-warning"
+                        [ngbTooltip]="
+                            result.participation!.exercise!.assessmentType !== AssessmentType.AUTOMATIC
+                                ? 'artemisApp.result.preliminaryTooltipSemiAutomatic'
```

Comment:
Wouldn't you need a translate here?
```suggestion
                                ? 'artemisApp.result.preliminaryTooltipSemiAutomatic' | translate
```

Reply:
I guess the result of the whole statement should be translated

Reply:
I changed it